src/types/response_types.ts

```diff
@@ -21,6 +21,12 @@ import {
 import U32 from '@polkadot/types/primitive/U32';
 import { AnyJson, Codec } from '@polkadot/types/types';
 
+export type AnyCodecAndJson =
+	| AnyJson
```

Comment:
Maybe this is a normal pattern in TS, feel free to accept or reject.
```suggestion
	AnyJson
```